app/javascript/components/pages/PatientInfo.js

```diff
@@ -45,20 +45,43 @@ export default class PatientInfo extends Component {
     } = this.state.patient;
     const { patient } = this.state;
```

Comment:
You're destructuing here (awesome) but no where else in the code. It's good practice for everyone to follow the same pattern. just FYI